<launch>

  <arg name="color_topic" default="kinect2/sd/image_color_rect" />
  <arg name="depth_topic" default="kinect2/sd/image_depth_rect" />

  <node pkg="kinect_tomato_searcher" type="kinect_tomato_searcher_node" name="search_tomato_node">
    <remap from="rgb" to="$(arg color_topic)" />
    <remap from="depth" to="$(arg depth_topic)" />
  </node>

  <node pkg="kinect_tomato_searcher" type="tomato_broadcaster" name="tomato_broadcaster">
    <remap from="tomato_point" to="tomato_point/raw" />
  </node>

</launch>
